runtime/lua/vim/lsp/builtin_callbacks.lua

```diff
@@ -260,7 +260,7 @@ BuiltinCallbacks['textDocument/definition'] = {
 
     local data_file = vim.uri_to_fname(data.uri)
 
-    if data_file ~= util.get_uri(current_file) then
+    if data_file ~= vim.fname_to_uri(current_file) then
```

Comment:
I suggested `vim.uri_from_fname` so that all URI-related functions were prefixed with `uri_`. That helps discoverability.

Reply:
I see. I'll change :)